.github/workflows/benchmark.yml

```diff
@@ -24,14 +24,14 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@stable
 
-      - run: cargo bench --bench schema_builder_bench
```

Comment:
Yes, because I wasn't clear on the purpose. Do we have that in order to have the benchmark results in the GitHub actions log? I'm happy to put that back in and add one for the new benchmarks.